### DISCUSSION TITLE
Anydesk Integration

### DISCUSSION THREAD

**Discussion by EliomarAraujo** [amidaware/tacticalrmm#1151] [General]
Has anyone managed to integrate Anydesk? I'm trying to no avail.

**Comment by silversword411** [comment]
?
https://docs.tacticalrmm.com/3rdparty_anydesk/

**Reply by EliomarAraujo** [reply]
Thanks for the answer. I did these steps, but it doesn't collect the remote ID automatically, I have to manually enter it in the machine's properties. is that right?

**Reply by wh1te909** [reply]
No, you need to setup a collector task to get the ID automatically. You can either use one of the community scripts that comes bundled with tactitcalrmm, or write your own. https://docs.tacticalrmm.com/functions/automated_tasks/#Collector%20Tasks

**Reply by EliomarAraujo** [reply]
thank you very much friend. I created the task and set it to manual, I have to run it manually to collect the ID.

**Reply by wh1te909** [reply]
you can set it to run on a schedule, doesn't have to be ran manually
